_topics/en/multisignature.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Multisignature
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - 2pECDSA
+  - Two-Party ECDSA (2pECDSA)
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Multisignatures** are digital signatures that look like they were
+  created using only a single private key but which were actually
+  created using two or more keys.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Only a single key and a single signature are published onchain when
```

Comment:
This paragraph is enumerating the two advantages of using multisignatures over multisig:

- size/fee savings
- privacy/fungibility

I think it might help to say that explicitly in the first sentence.